Docker/entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+cat > .env  <<-EOF
+APP_KEY=${APP_KEY:-SomeRandomKey!}
+APP_ENV=${APP_ENV:-local}
+APP_DEBUG=${APP_ENV:-true}
+APP_LOCALE=${APP_LOCALE:-en}
+APP_FALLBACK_LOCALE=${APP_FALLBACK_LOCALE:-en}
+
+DB_CONNECTION=${DB_CONNECTION:-mysql}
+DB_HOST=${DB_PORT_3306_TCP_ADDR:-localhost}
```

Comment:
what if you want to override it and maybe use your own mariadb/mysql server container which does not provide this ENV_VAR?